src/test/java/integrationTest/CartItemsTest.java

```diff
@@ -0,0 +1,198 @@
+        Resource resource = new ClassPathResource("/test.properties");
+        Properties properties = PropertiesLoaderUtils.loadProperties(resource);
+
+        host = properties.getProperty("host");
+        loginRoute = properties.getProperty("loginRoute");
+        getCartItemsRoute = properties.getProperty("getCartItemsRoute");
+        addCartItemsRoute = properties.getProperty("addCartItemsRoute");
+        deleteCartItemsRoute = properties.getProperty("deleteCartItemsRoute");
+        userGetUserRoute = properties.getProperty("userGetUserRoute");
+
+        getJwtToken();
+        getUsersEntity();
+    }
+
+    public void getJwtToken() {
+        String url = host + loginRoute;
+        testUsersEntity.setUserName("testuser");
+        testUsersEntity.setPassword("newPassword");
+
+        Response response = client
+                .target(url)
+                .request(MediaType.APPLICATION_JSON)
+                .post(Entity.entity(testUsersEntity, MediaType.APPLICATION_JSON));
+        jwtModel = response.readEntity(JwtModel.class);
+
+        if (response.getStatus() != 200) {
+            log.error("no jwtToken");
+            throw new SkipException("");
+        }
+    }
+
+    public void getUsersEntity() {
+        String url = host + userGetUserRoute;
+
+        Response response = client
+                .target(url)
+                .request(MediaType.APPLICATION_JSON)
+                .header("Authorization", "Bearer " + jwtModel.getJwt())
+                .get();
+
+        testUsersEntity = response.readEntity(UsersEntity.class);
+
+        if (response.getStatus() != 200) {
+            log.error("no user");
+            throw new SkipException("");
+        }
+    }
+
+    @Test(dataProvider = "getCSVDataAddCartItems")
+    public void addCartItems(String testCaseId) {
+        String url = host + addCartItemsRoute;
+        Properties prop = TestUtil.getCSVData(testCSVFile, testCaseId);
+        cartItemsDTO = new CartItemsDTO(Integer.parseInt(prop.getProperty("productId")), Integer.parseInt(prop.getProperty("quantity")));
+
+        Response response = client
+                .target(url)
+                .request(MediaType.APPLICATION_JSON)
+                .header("Authorization", "Bearer " + jwtModel.getJwt())
+                .post(Entity.entity(cartItemsDTO, MediaType.APPLICATION_JSON));
+
+        if (testCaseId.equalsIgnoreCase("getCSVDataAddCartItemsSuccess")) {
+            assertEquals(response.getStatus(), 200);
+        } else {
+            assertNotEquals(response.getStatus(), 200);
+        }
+    }
+
+    @Test(dependsOnMethods = {"addCartItems"})
+    public void getCartItems() {
+        String url = host + getCartItemsRoute;
+        Response response = client
+                .target(url)
+                .queryParam("id", testUsersEntity.getId())
+                .request(MediaType.APPLICATION_JSON)
+                .header("Authorization", "Bearer " + jwtModel.getJwt())
```

Comment:
Instead of using "Authorization" string multiple times, use a pre-defined enum HttpHeader.AUTHORIZATION from Spring